### ISSUE TITLE
Databricks and Trino are missing from the data source type dropdown in the development settings page after migrating to Universal SQL

### ISSUE THREAD

**Issue by ycaffee** [evidence-dev/evidence#1443]
## Context
I recently used the VSCode extension to migrate a small Evidence project to use Universal SQL i.e. updated to Evidence version 24.0+. I am having trouble connecting to my data in Databricks (and I believe Trino has the same issue). 

## Steps To Reproduce

1. Navigate to http://localhost:3000/settings/
2. In the Data Sources section, click the "Add new source" button
3. Click on the "Datasource Type" dropdown
4. There should be a data source listed for each of the data sources in `evidence.plugins.yaml` and `package.json`

## Expected Behavior

Databricks and Trino should be listed in the data source type dropdown at http://localhost:3000/settings/

## Actual Behaviour

Databricks and Trino are missing from the data source type dropdown at http://localhost:3000/settings/.

### Ruling out misconfiguration
I understand that the data sources listed at http://localhost:3000/settings/ depend on the contents of my `evidence.plugins.yaml` and `package.json` files. 

Both the Databricks and Trino packages are installed and `npm list` shows
```
├── @evidence-dev/bigquery@2.0.1
├── @evidence-dev/core-components@3.0.4
├── @evidence-dev/csv@1.0.3
├── @evidence-dev/databricks@1.0.1
├── @evidence-dev/duckdb@1.0.3
├── @evidence-dev/evidence@24.0.7
├── @evidence-dev/mssql@1.0.1
├── @evidence-dev/mysql@1.0.1
├── @evidence-dev/postgres@1.0.1
├── @evidence-dev/snowflake@1.0.1
├── @evidence-dev/sqlite@2.0.1
└── @evidence-dev/trino@1.0.1
```

My `evidence.plugins.yaml` file contains
```
components:
  "@evidence-dev/core-components": {}

datasources:
  "@evidence-dev/bigquery": { }
  "@evidence-dev/csv": { }
  "@evidence-dev/databricks": { }
  "@evidence-dev/duckdb": { }
  "@evidence-dev/mssql": { }
  "@evidence-dev/mysql": { }
  "@evidence-dev/postgres": { }
  "@evidence-dev/snowflake": { }
  "@evidence-dev/sqlite": { }
  "@evidence-dev/trino": { }
 ```

## Possible root cause
Are `node_modules/@evidence-dev/databricks/package.json` and `node_modules/@evidence-dev/trino/package.json`
 missing the "evidence" attribute/configuration? I noticed the package.json's for other data sources contain something like:
```
{
  "name": "@evidence-dev/sqlite",
 ...
  "evidence": {
    "datasources": [
      "sqlite"
    ],
    "icon": "Sqlite"
  },
...
}
```

### Environment

- Node version (`node -v`): v18.17.0
- npm version (`npm -v`): 9.6.7
- OS: macOS v. 12.6
- Browser: Brave
- Database: Databricks

**Comment by ItsMeBrianD** [comment]
This is resolved by #1444

**Comment by archiewood** [comment]
Tested this out in example project getting the following error.

I suspect that it's the databricks token 

![CleanShot 2024-01-03 at 17 43 34@2x](https://github.com/evidence-dev/evidence/assets/58074498/89c48157-8f8c-4c16-b884-509c4accb1d5)

**Comment by ycaffee** [comment]
Thanks @archiewood and @ItsMeBrianD. These changes worked for me when I applied them locally. 
<img width="993" alt="Screen Shot 2024-01-03 at 1 58 28 PM" src="https://github.com/evidence-dev/evidence/assets/101021648/453b0254-d337-4dba-a62a-17e3fd31e303">

**Comment by archiewood** [comment]
Closed by #1444. Will be available after next release and update to template project
